routes/users.js

```diff
@@ -7,6 +7,7 @@ const userValidator = require('../middlewares/validators/user')
 router.post('/', authenticate, userValidator.createUser, usersController.addNewUser)
 router.patch('/:id', authenticate, userValidator.updateUser, usersController.updateUser)
 router.get('/', authenticate, usersController.getUsers)
+router.get('/self', authenticate, usersController.getSelfDetails)
```

Comment:
@Rucha1499 , can you add the API documentation annotations over this route?
Take reference from healthCheck.js

Reply:
Done the changes regarding the returning of error. Added the API documentation annotations over the route users/self.